reference/wishlists.v3.yml

```diff
@@ -437,7 +446,7 @@ paths:
         '500':
           description: Internal server error.
       summary: Add Wishlist Item
```

Comment:
```suggestion
      summary: Add wishlist item
```

Reply:
Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.